docker-compose.yml

```diff
@@ -195,11 +228,31 @@ services:
       - "8080"
     depends_on:
       - keycloak-db
+  pgadmin:
```

Comment:
Note to myself: Document pgadmin and credentials in the Wiki before merging this PR